Orm/Xtensive.Orm/Reflection/AttributeHelper.cs

```diff
@@ -1,54 +1,33 @@
-// Copyright (C) 2008-2020 Xtensive LLC.
+// Copyright (C) 2008-2021 Xtensive LLC.
 // This code is distributed under MIT license terms.
 // See the License.txt file in the project root for more information.
 // Created by: Alex Yakunin
 // Created:    2008.06.13
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Linq;
 using System.Reflection;
-using Xtensive.Collections;
-using Xtensive.Core;
-
 
 namespace Xtensive.Reflection
 {
+  using AttributesKey = ValueTuple<MemberInfo, Type, AttributeSearchOptions>;
+  using PerAttributeKey = ValueTuple<MemberInfo, AttributeSearchOptions>;
+
```

Comment:
The file contains only one namespace declaration so no reason to have ```using``` inside namespace, they should be moved up outside.

Reply:
Outside `using` for types is more verbose, but Ok. Moved